src/display/container/init.go

```diff
@@ -0,0 +1,307 @@
+	page.NetChart.BarWidth = 9
+	page.NetChart.BarColors = []ui.Color{ui.ColorGreen, ui.ColorCyan}
+	page.NetChart.LabelStyles = []ui.Style{ui.NewStyle(ui.ColorClear)}
+	page.NetChart.NumStyles = []ui.Style{ui.NewStyle(ui.ColorBlack)}
+
+	// Intialise Bar Chart for Blk Chart
+	page.BlkChart.Data = []float64{0, 0}
+	page.BlkChart.Title = " Blk I/O "
+	page.BlkChart.Labels = []string{"Read", "Write"}
+	page.BlkChart.BorderStyle.Fg = ui.ColorCyan
+	page.BlkChart.TitleStyle.Fg = ui.ColorClear
+	page.BlkChart.BarWidth = 9
+	page.BlkChart.BarColors = []ui.Color{ui.ColorGreen, ui.ColorCyan}
+	page.BlkChart.LabelStyles = []ui.Style{ui.NewStyle(ui.ColorClear)}
+	page.BlkChart.NumStyles = []ui.Style{ui.NewStyle(ui.ColorBlack)}
+
+	// Initialize Table for Container Details Table
+	page.DetailsTable.Title = " Details "
+	page.DetailsTable.BorderStyle.Fg = ui.ColorCyan
+	page.DetailsTable.TitleStyle.Fg = ui.ColorClear
+	page.DetailsTable.ColResizer = func() {
+		x := page.DetailsTable.Inner.Dx()
+		page.DetailsTable.ColWidths = []int{
+			x / 2,
+			x / 2,
+		}
+	}
+
+	// Initialize Table for Mount Table
+	page.MountTable.Title = " Mounts "
+	page.MountTable.BorderStyle.Fg = ui.ColorCyan
+	page.MountTable.TitleStyle.Fg = ui.ColorClear
+	page.MountTable.ColGap = 1
+	page.MountTable.ColResizer = func() {
+		x := page.MountTable.Inner.Dx()
+		page.MountTable.ColWidths = []int{
+			4 * x / 10,
+			4 * x / 10,
+			2 * x / 10,
+		}
+	}
+	page.MountTable.Header = []string{"SRC", "DST", "Mode"}
+	page.MountTable.CursorColor = ui.ColorCyan
+
+	// Initialize Table for Network Table
+	page.NetworkTable.Title = " Networks "
+	page.NetworkTable.BorderStyle.Fg = ui.ColorCyan
+	page.NetworkTable.TitleStyle.Fg = ui.ColorClear
+	page.NetworkTable.ColGap = 1
+	page.NetworkTable.ColResizer = func() {
+		x := page.NetworkTable.Inner.Dx()
+		page.NetworkTable.ColWidths = []int{
+			2 * x / 10,
+			2 * x / 10,
+			3 * x / 10,
+			x / 10,
+		}
+	}
+	page.NetworkTable.Header = []string{"Name", "Driver", "IP", "Ingress"}
+	page.NetworkTable.CursorColor = ui.ColorCyan
+
+	// Initialize Table for CPU Usage Table
+	page.CPUUsageTable.Title = " Per CPU "
+	page.CPUUsageTable.BorderStyle.Fg = ui.ColorCyan
+	page.CPUUsageTable.TitleStyle.Fg = ui.ColorClear
+	page.CPUUsageTable.ColResizer = func() {
+		x := page.CPUUsageTable.Inner.Dx()
+		page.CPUUsageTable.ColWidths = []int{
+			x / 2,
+			x / 2,
+		}
+	}
+	page.CPUUsageTable.Header = []string{"CPU", "Usage"}
+	page.CPUUsageTable.CursorColor = ui.ColorCyan
+
+	// Initialize Table for Port Map Table
+	page.PortMapTable.Title = " Port Mappings "
+	page.PortMapTable.BorderStyle.Fg = ui.ColorCyan
+	page.PortMapTable.TitleStyle.Fg = ui.ColorClear
+	page.PortMapTable.ColResizer = func() {
+		x := page.PortMapTable.Inner.Dx()
+		page.PortMapTable.ColWidths = []int{
+			3 * x / 10,
+			2 * x / 10,
+			2 * x / 10,
+			2 * x / 10,
+		}
+	}
+	page.PortMapTable.Header = []string{"IP", "Host", "Container", "Type"}
```

Comment:
Small nit, but can you change `IP` to `HostIP`? Just to be as clear as possible and consistent with the docker API.